sympy/testing/runtests.py

```diff
@@ -2030,7 +2057,7 @@ def findout_terminal_width():
                 process = subprocess.Popen(['stty', '-a'],
                     stdout=subprocess.PIPE,
                     stderr=subprocess.PIPE)
-                stdout = process.stdout.read()
+                stdout, stderr = process.communicate()
```

Comment:
Similar to using `with open` this is needed to ensure that the subprocess completes rather than becoming a dangling zombie process.